Loki windows amd64 release binaries

Would like to run Loki on a Windows Server / window amd64. Cannot find compatible binaries windows release. Can you provide directions on how to make windows build of Loki or provide the zip archive of the binaries on the release tab.
Thanks

Loki process fails with invalid memory address or dereferenced pointer. when initializing the processes storage. 


```
level=info ts=2019-08-20T14:35:30.8571574Z caller=loki.go:125 msg=initialising module=server
level=info ts=2019-08-20T14:35:30.8591573Z caller=server.go:120 http=[::]:80 grpc=[::]:9095 msg="server listening on addresses"
level=info ts=2019-08-20T14:35:30.863176Z caller=loki.go:125 msg=initialising module=overrides
level=info ts=2019-08-20T14:35:30.863176Z caller=override.go:48 msg="per-tenant overides disabled"
level=info ts=2019-08-20T14:35:30.863176Z caller=loki.go:125 msg=initialising module=table-manager
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x38 pc=0x10aac91]

goroutine 1 [running]:
github.com/grafana/loki/vendor/github.com/cortexproject/cortex/pkg/chunk/storage.NewTableClient(0x14db233, 0x3, 0x0, 0x4000000000000000, 0x4024000000000000, 0x0, 0x0, 0x0, 0x186a0, 0x3ff4cccccccccccd, ...)
        C:/gopath/src/github.com/grafana/loki/vendor/github.com/cortexproject/cortex/pkg/chunk/storage/factory.go:160 +0x81
github.com/grafana/loki/pkg/loki.(*Loki).initTableManager(0xc0001b6000, 0xc00007c4b0, 0x7)
        C:/gopath/src/github.com/grafana/loki/pkg/loki/modules.go:201 +0x11c
github.com/grafana/loki/pkg/loki.(*Loki).initModule(0xc0001b6000, 0x7, 0x0, 0x0)
        C:/gopath/src/github.com/grafana/loki/pkg/loki/loki.go:127 +0x234
github.com/grafana/loki/pkg/loki.(*Loki).init(0xc0001b6000, 0x8, 0xc0004e66b0, 0x10)
        C:/gopath/src/github.com/grafana/loki/pkg/loki/loki.go:116 +0x82
github.com/grafana/loki/pkg/loki.New(0x8, 0x1, 0x14dbc28, 0x4, 0x0, 0x0, 0x50, 0x0, 0x0, 0x2387, ...)
        C:/gopath/src/github.com/grafana/loki/pkg/loki/loki.go:86 +0x15a
main.main()
        C:/gopath/src/github.com/grafana/loki/cmd/loki/main.go:69 +0x380
```

Command to run process: From a PowerShell console.

```
$dockerCommand = 'docker run loki:v0.3.0-nanoserver-1809 --% -auth.enabled false -server.http-listen-port 3100 -ingester.lifecycler.addr 127.0.0.1   -ring.store inmemory -distributor.replication-factor 1   -ingester.final-sleep 0 -ingester.chunks-idle-period 5m   -ingester.chunks-retain-period 30s -config-yaml schema_config:configs:-from:2018-04-15store:boltdbobject_store:filesystemschema:v9index:prefix:index_period:168h -boltdb.dir "C:\ProgramData\loki\index" -local.chunk-directory "C:\ProgramData\loki\chunk" -validation.enforce-metric-name false   -validation.reject-old-samples true -validation.reject-old-samples.max-age 168h -store.max-look-back-period 0 -dynamodb.chunk-table.inactive-read-throughput 0 -dynamodb.chunk-table.inactive-write-throughput 0 -dynamodb.chunk-table.read-throughput.scale.max-capacity 0 -dynamodb.chunk-table.write-throughput.scale.max-capacity 0 -dynamodb.periodic-table.inactive-read-throughput 0 -dynamodb.periodic-table.inactive-write-throughput 0 -dynamodb.periodic-table.read-throughput.scale.max-capacity 0 -dynamodb.periodic-table.write-throughput.scale.max-capacity 0 -table-manager.retention-deletes-enabled false -table-manager.retention-period 0'
iex $dockerCommand 

```

--- a/Makefile
+++ b/Makefile
@@ -194,7 +194,7 @@ cmd/promtail/promtail-debug: $(APP_GO_FILES) pkg/promtail/server/ui/assets_vfsda
 # concurrency is limited to 4 to prevent CircleCI from OOMing. Sorry
 GOX = gox $(GO_FLAGS) -parallel=4 -output="dist/{{.Dir}}-{{.OS}}-{{.Arch}}" -arch="amd64 arm64 arm" -os="linux"
 dist: clean
-	CGO_ENABLED=0 $(GOX) ./cmd/loki
+	CGO_ENABLED=0 $(GOX) -osarch="windows/amd64" ./cmd/loki
 	CGO_ENABLED=0 $(GOX) -osarch="darwin/amd64 windows/amd64 freebsd/amd64" ./cmd/promtail ./cmd/logcli
 	gzip dist/*
 	pushd dist && sha256sum * > SHA256SUMS && popd

